popupwindow.js

```diff
@@ -0,0 +1,121 @@
+		also the leap into electronic typesetting, remaining essent`,
+  },
+
+  {
+    heading: "Multi-post <br/>Stories",
+    image: "./imagsPop/multi1.png",
+    content: `Lorem Ipsum is simply dummy text of the printing 
+		and typesetting industry. Lorem Ipsum has been the industry's 
+		standard dummy text ever since the 1500s, when an unknown 
+		printer took a galley of type and scrambled it to make a type 
+		specimen book. It has survived not only five centuries, but 
+		also the leap into electronic typesetting, remaining essent`,
+  },
+
+  {
+    heading: "Facebook",
+    image: "./imagsPop/facebook1.png",
+    content: `Lorem Ipsum is simply dummy text of the printing 
+		and typesetting industry. Lorem Ipsum has been the industry's 
+		standard dummy text ever since the 1500s, when an unknown 
+		printer took a galley of type and scrambled it to make a type 
+		specimen book. It has survived not only five centuries, but 
+		also the leap into electronic typesetting, remaining essent`,
+  },
+
+  {
+    heading: "Multi-post <br/>Stories",
+    image: "./imagsPop/project image 2.png",
+    content: `Lorem Ipsum is simply dummy text of the printing 
+		and typesetting industry. Lorem Ipsum has been the industry's 
+		standard dummy text ever since the 1500s, when an unknown 
+		printer took a galley of type and scrambled it to make a type 
+		specimen book. It has survived not only five centuries, but 
+		also the leap into electronic typesetting, remaining essent`,
+  },
+];
+
+const modal = document.querySelector(".modal");
+const popupWrap = document.querySelector(".pop-up-wrap");
+const projectBtn = document.querySelectorAll(".see-btn");
+
+projectBtn.forEach(function (Btn) {
+  Btn.addEventListener("click", function (e) {
+    const i = e.currentTarget.dataset.id;
+    popupWrap.innerHTML = `<div class="pop-up-nav">
+		  <h2 class="poppins" id="logo">${dataPopover[i].heading}</h2>
+		  <span class = "pop-up-closeBtn">&times;</span>
+		</div>
+	  
+	  <div class="pop-up-about">
+		  <ul class="poppins canopy-modal">
+			<li class="about-li" id="canopy">CANOPY</li>
+			<li class="about-li">Back End Dev</li>
+			<li class="about-li">2015</li>
+		  </ul>
+	  </div>
+	  
+	  <div class="pop-up-image-wrap">
+		<img class="snapshoot" src="${dataPopover[i].image}" alt="Tanner Christensen's portfolio image">
+	  </div>
+		 
+	  <div class="pop-up-text-content">
+		<p class="poppins-regular" id="modal-text">${dataPopover[i].content}</p>
+		<div class="tag-wrap">
+		<ul class="poppins-regular language-button">
+		  <li class="languages">html</li>
+		  <li class="languages">css</li>
+		  <li class="languages">javascript</li>
+		</ul>
+	  <div class="pop-up-btn">
+		<button class="poppins-regular see-live-btn">See live<img class="live-icon" src="./imagsPop/see-live-icon.svg"/></button>
+		<button class="poppins-regular see-source-btn">See source<img id="github-modal-icon" src="./images/Vector(2).svg"/></button>
```

Comment:
- [ ] it would be better if you add effects to your button as in the Figma design (hover, pressed...etc) in order to match the Figma design 
<img width="577" alt="Screen Shot 2022-10-19 at 18 03 04" src="https://user-images.githubusercontent.com/61790587/196757384-91d036d5-5926-4850-9d88-e9a6ae85fd7b.png">